src/Database/Log/LoggingStatement.php

```diff
@@ -109,15 +109,37 @@ public function bindValue($column, $value, $type = 'string')
      * Sets the logger object instance. When called with no arguments
      * it returns the currently setup logger instance
      *
+     * @deprecated 3.5.0 Use getLogger() and setLogger() instead.
```

Comment:
I agree, before there are no clear sniffers/guidelines it is not worth it.
And I agree, the order it currently is is fine.
```
@other
@param
@throws
@return
```
usually